storage/innobase/row/row0upd.cc

```diff
@@ -3246,16 +3216,10 @@ static bool row_upd_check_autoinc_counter(const upd_node_t *node, mtr_t *mtr) {
 
   index = node->table->first_index();
 
-<<<<<<< HEAD
-  referenced = row_upd_index_is_referenced(index, trx);
+  auto referenced = row_upd_index_is_referenced(index);
 #ifdef WITH_WSREP
   bool foreign = wsrep_row_upd_index_is_foreign(index, thr_get_trx(thr));
```

Comment:
we can use trx here instead of thr_get_trx()

Reply:
Done